docs/runbooks/SSPHighRateRejectedVms.md

```diff
@@ -0,0 +1,24 @@
+# SSPHighRateRejectedVms
+
+## Meaning
+
+This alert indicates that many Virtual Machines were not valid and therefore were not created or not modified.
+This means either a user or a script is continuously trying to create invalid Virtual Machines.
+
+## Impact
+
+There is no immediate impact, however one should see what is the reason for the large amount of rejected Virtual Machines since it may mean that something is not going to work as planned (for example a script may be consistently failing to create or modify Virtual Machines)
```

Comment:
@borod108  I think it would be better to slip to 2 sentences.
There is no immediate impact, however one should see what is the reason for the large amount of rejected Virtual Machines. 
Large amount of rejected Virtual Machines means that something is not going to work as planned, for example, a script may be consistently failing to create or modify Virtual Machines.

Reply:
done